x/nft/keeper/class.go

```diff
@@ -0,0 +1,61 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/nft"
+)
+
+// NewClass defines a method for creating a new nft class
+func (k Keeper) NewClass(ctx sdk.Context, class nft.Class) error {
+	if k.HasClass(ctx, class.Id) {
+		return sdkerrors.Wrap(nft.ErrClassExists, class.Id)
+	}
+	bz := k.cdc.MustMarshal(&class)
```

Comment:
Let's avoid using `Must*` - this function panics. Instead, let's use `Marshal` and return errors, especially that `NewClass` retrurns error.